src/Resources/Resources/help/New-AzPolicyAssignment.md

```diff
@@ -276,7 +276,7 @@ Accept wildcard characters: False
 ```
 
 ### -Name
-Specifies a name for the policy assignment.
+Specifies a name for the policy assignment. Limited to **24** characters
```

Comment:
Hi @mentat9 , could you kindly help us to confirm the limit length? I cannot find the logic in azure-powershell code.

Reply:
Turns out it's 64 characters for subscription, resource group, and resource scopes. 24 characters for management group scopes. All of that notwithstanding, I'm not convinced that information belongs in this message. If the name is invalid for any reason (including length), the error message that comes from the backend tells what is wrong with the name and what's needed to fix it. Since there is no name validation in the cmdlet, I think it's better to rely on the information in the backend validation failure message. Adding it here means that it will be wrong if the validation rules in the backend change.

@michawets, please add any thoughts you have.

Reply:
Thanks a lot for your information! @mentat9

Reply:
Hi @michawets , thanks a lot for your contribution! But we think that relying on the information from backend is a better choose. Would you mind to close this issue? We still thanks very much for your contribution and sorry for this.

Reply:
Hi @wyunchi-ms , I understand the reason not to allow it. I reason is that we use this in a CI/CD pipeline, and it would have been nice to know this limit before running it and see it in the docs😉